src/resources/views/character/mail-timeline.blade.php

```diff
@@ -77,7 +76,30 @@
             @endforeach
 
           </li>
-          @endif
+
+        @endif
+
+        @if($message->recipients->where('recipient_type', 'mailing_list')->count() > 0)
+
+          <li>
+            <b>{{ trans('web::seat.to_mailing_list') }}:</b>
+
+            @foreach($message->recipients->where('recipient_type', 'mailing_list') as $recipient)
+
+              @if (! is_null($recipient->mailing_list))
+                {{ $recipient->mailing_list->name }}
+              @else
+                {{ trans('web::seat.unknown') }}
+              @endif
+
+            @endforeach
+
+          </li>
+
+        @endif
+
+        <b>{{ trans('web::seat.source') }}: </b>
```

Comment:
Is this source the character that sent to the mailing list? Why don't we just update the From with that?

Reply:
it's the data source is not equal to the sender. As the source might be either sender or receiver. The main reason to include this information are that HR scans the Mail-timeline and might be interested which of all the characters they have access to had provided the mail-information as they might be in corporation which acts as receiver.

Reply:
in the PR's picture you can see how 'from' 'to' and 'source' can be totally different.

Reply:
Is the source the account that actually has this email?

Reply:
yes 
![image](https://user-images.githubusercontent.com/6583519/50771392-d2076e80-128a-11e9-9a36-4bf47ecb498e.png)

Reply:
We need a different name than source. I also want to suggest; as this is a meta-like piece of information, maybe it should move to the right under the new title and not be part of the headers section but rather timestamps.

Lastly, did you check that we won't have a case where two people on the same SeAT instance, with the same email wont disclose the presence of these accounts on the instance with this feature?

Reply:
I moved the source to the right below the timestamp:
![image](https://user-images.githubusercontent.com/6583519/51075280-ba185c00-1689-11e9-8d21-133190955b52.png)
 
Until you have better suggestion i stay with source as name, as it appears to be a good terminology in regards to mail transactions. From or origin may be misleading.

Regarding the exposing of characters and corporations on same seat instance: this is already a fact in not only this blade but any Datatable be it wallet transactions, intel page, contacts etc. In this blade `from` and `to` already expose the existance. IMHO this should not block this PR.

Furthermore as source is the character_id attribute from the table: The source of an email, to an auth user must always be affiliated with the auth()->user because this is quaranteed in the service:
https://github.com/eveseat/services/blob/ffba218379d7aff0e8fec52b8c23c0a4a03fb3cb/src/Repositories/Character/Mail.php#L141